build/Build.md

```diff
@@ -0,0 +1,190 @@
+# 1. Building a container image
+
+This chapter depicts important information about building your first WSO2 container image.
+
+### Contents
+
+The following will be discussed in detail in the document.
+
+* [1.1. Why should you choose Docker as your container runtime ?](#Why-docker)
```

Comment:
@DilanUA these links do not work. Can you please check?

Reply:
Will fix in next PR.